dev/breeze/src/airflow_breeze/params/shell_params.py

```diff
@@ -522,7 +522,14 @@ def env_variables_for_docker_commands(self) -> dict[str, str]:
         _set_var(_env, "AIRFLOW__CELERY__BROKER_URL", self.airflow_celery_broker_url)
         _set_var(_env, "AIRFLOW__CORE__AUTH_MANAGER", self.auth_manager_path)
         _set_var(_env, "AIRFLOW__CORE__EXECUTOR", self.executor)
-        _set_var(_env, "AIRFLOW__CORE__SIMPLE_AUTH_MANAGER_USERS", "admin:admin,viewer:viewer")
+        if self.auth_manager == FAB_AUTH_MANAGER:
+            _set_var(
+                _env,
+                "AIRFLOW__DATABASE__EXTERNAL_DB_MANAGERS",
+                "airflow.providers.fab.auth_manager.models.db.FABDBManager",
+            )
```

Comment:
```suggestion
```

I don't think we need this any longer, now that the FAB auth manager is able to add it's own db manager.